make/tools.mk

```diff
@@ -101,8 +100,9 @@ bin/scratch/tools/cosign_$(COSIGN_VERSION)_$(HOST_OS)_$(HOST_ARCH): | bin/scratc
 # ginkgo #
 ##########
 
-bin/tools/ginkgo: | bin/tools
-	GOBIN=$(shell pwd)/$(dir $@) go install github.com/onsi/ginkgo/ginkgo@$(GINKGO_VERSION)
+# We don't speicify a version number here because we want to use the version in cert-manager's go.mod
```

Comment:
nit: specify 😅

Reply:
good spot, sorry about that!

fixed now :ok_hand: